pkg/clusteragent/admission/patch/patch_request.go

```diff
@@ -47,17 +69,11 @@ type Request struct {
 	LibConfig common.LibConfig `json:"lib_config"`
 
 	// Target k8s object
-	K8sTarget K8sTarget `json:"k8s_target"`
+	K8sTarget *K8sTarget `json:"k8s_target_v2,omitempty"`
```

Comment:
This struct contains the request that we receive from remote config, so I guess this needs to be keep in sync with the structures managed by remote config? Is there a reference that we can add here to the definition so that we don't forget?